legacy_ibc_testing/testing/chain.go

```diff
@@ -494,6 +494,10 @@ func (chain *TestChain) CreateTMClientHeader(chainID string, blockHeight int64,
 	vsetHash := tmValSet.Hash()
 	nextValHash := nextVals.Hash()
 
+	if tmValSet == nil {
+		panic("tmValSet cannot be nil")
+	}
+
```

Comment:
can panic very early before 
```
	var (
		valSet      *tmproto.ValidatorSet
		trustedVals *tmproto.ValidatorSet
	)
	```

Reply:
@yaruwangway I didn't get this? Can you elaborate?

Reply:
Yea I'm happy to address anything actionable from this comment. For now I'll merge this PR as it's intended to be pretty trivial, just to make linters pass